### DISCUSSION TITLE
Single quote doesn't decode in old helper function.

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [laravel/framework#44960] [Q&A]
I have value in a database like this "29 O'Dea Street"

when using the old helper function in input field.

```php
<x-adminlte-input label="Address 1" id="address_1" name="address_1" :value="old('address_1', $order->billing->address_1)" />
```

Its output looks like this
![Screenshot](https://user-images.githubusercontent.com/3838722/202090001-0c0cb5f4-dd35-4769-af1f-5d886ebacf60.png)

**Comment by chuckrincon** [comment]
Hi ✋🏻 
You could try using the [escaped output syntax _{!! !!}_.](https://laravel.com/docs/9.x/blade#displaying-unescaped-data)
```blade
<x-adminlte-input :value="old('address_1', '{!! $order->billing->address_1 !!}')" />
```

**Reply by shankhadevpadam** [reply]
@chuckrincon I fixed my self.
